static/js/click_handlers.js

```diff
@@ -541,7 +541,7 @@ export function initialize() {
 
     // SIDEBARS
 
-    $(".right-sidebar .login_button").on("click", (e) => {
+    $(".column-right .login_button").on("click", (e) => {
```

Comment:
Should this be just `$(body).on("click", ".login_button") {` instead? 

It doesn't seem like it should be important what part of the app contains this kind of button, or whether the button was rendered to the DOM during app startup or when this handler is opened...

Reply:
Yeah!